src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,176 @@
+package duke;
+
+import java.io.IOException;
+
+/**
+ * Duke is a task management assistant. It accepts user commands with the appropriate parameters and time
+ * information (if any). It can add/delete tasks, mark tasks as done, and list all tasks. Upon exit, the
+ * tasks will be saved to a data file. 
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public static void main(String[] args) {
+        new Duke("data/tasks.txt").runCli();
+    }
+
+    /**
+     * Constructs a Duke object. An attempt will be made to load in tasks from the default file. If the file
+     * does not exist or cannot be accessed, an empty task list will be loaded.
+     */
+    public Duke() {
```

Comment:
Perhaps instead of having two constructors with identical implementation, you can get this constructor to call the other with "data/tasks.txt" as the argument instead?